CONTRIBUTING.md

```diff
@@ -59,6 +59,22 @@ During the collector release process, all `./.chloggen/*.yaml` files are transcr
 
 Alternately, copy `./.chloggen/TEMPLATE.yaml`, or just create your file from scratch.
 
+## Platform Agnostic Code
+
+In order to ensure compatibility with different operating systems, code should be platform-agnostic. Below are some guidelines to follow when writing platform-agnostic code:
+
+* Avoid using platform-specific libraries, features etc. Please opt for cross-platform solutions. 
```

Comment:
This doesn't seem to say anything of substance.  What is the actionable recommendation here?  What is a `cross-platform` solution?  Is a portable multi-platform solution that uses different implementations per platform acceptable?  That would not be platform agnostic.